tests/query/parser/test_parser.py

```diff
@@ -40,7 +40,7 @@ def test_mql() -> None:
             "orderby": "ASC",
             "granularity": 60,
             "interval": None,
-            "with_totals": "True",
+            "with_totals": "False",
```

Comment:
These tests are setting `with_totals` to `False`. Do we need to add tests in `group by` to verify and assert that `with_totals` is set to `True`?

Reply:
There are a few implicit tests that check for this in `test_metrics_mql_api.py`. I added a more explicit test here: https://github.com/getsentry/snuba/pull/6218/commits/316cdc623caee95be82de73b903e558db3d5e05b

Reply:
Thanks for the clarification!